### PR TITLE
Update & Shortened install.ps1

### PR DESCRIPTION
**Pull request by tijb**
There was a few quotes symbols that were `“` instead of `"`.
Also updated the script to make it shorter.

### CODE REVIEW

File: bin/installer.ps1 [hunk 1]
```diff
@@ -1,66 +1,30 @@
-$DEIN_VIM_REPO = "https://github.com/Shougo/dein.vim"
+param(
+    [Parameter(Mandatory=$true)][String]$PluginDir
```
Comment (tijb) [inline bin/installer.ps1:2]: this makes the first parameter mandatory.

File: bin/installer.ps1 [hunk 2]
```diff
@@ -1,66 +1,30 @@
...
 }
 
-Write-Output ""
-
 # check git command
-if (!(Test-CommandExists git)) {
-    Write-Output 'Please install git or update your path to include the git executable!'
-    exit 1
+if (!(Get-Command git -ErrorAction SilentlyContinue -OutVariable $_)) {
+    throw 'Please install git or update your path to include the git executable!'
```
Comment (tijb) [inline bin/installer.ps1:18]: throw exits with an error.

File: bin/installer.ps1 [hunk 3]
```diff
@@ -1,66 +1,30 @@
...
 # make plugin dir and fetch dein
-if (!(Test-Path $INSTALL_DIR)) {
-    Write-Output "Begin fetching dein..."
-    New-Item $INSTALL_DIR -ItemType Directory | Out-Null
-    git clone $DEIN_VIM_REPO $INSTALL_DIR
-    Write-Output "Done."
-    Write-Output ""
-}
+Write-Output "Begin fetching dein..."
+git clone $DeinVimRepo $InstallDir
```
Comment (tijb) [inline bin/installer.ps1:24]: dont need to make the directory ourselves, git will make it.

File: bin/installer.ps1 [hunk 4]
```diff
@@ -1,66 +1,30 @@
...
+Write-Output "Done.`n"
 
-Write-Output "Please add the following settings for dein to the top of your vimrc (Vim) or init.vim (NeoVim) file:"
+Write-Host -ForegroundColor Yellow "Please add the following settings for dein to the top of your vimrc (Vim) or init.vim (NeoVim) file:"
```
Comment (tijb) [inline bin/installer.ps1:27]: made the post install comment stand out more.

File: bin/installer.ps1 [hunk 5]
```diff
@@ -70,15 +34,15 @@ Write-Output "  set nocompatible               `" Be iMproved"
 Write-Output "endif"
 Write-Output ""
 Write-Output "`" Required:"
-Write-Output "set runtimepath+=$INSTALL_DIR"
+Write-Output "set runtimepath+=$InstallDir"
```
Comment (tijb) [inline bin/installer.ps1:37]: nitpick: renamed the variables to follow Powershell conventions.

**Comment by Shougo** [comment]
Thanks.

**Pull request merged**